modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -5,12 +5,21 @@
 require_relative '../../../models/mobile/v0/adapters/claims_overview_errors'
 require_relative '../../../models/mobile/v0/claim_overview'
 require 'sentry_logging'
+require 'prawn'
 
 module Mobile
   module V0
     class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
+      TMP_BASE_PATH = Rails.root.join 'tmp', 'uploads', 'cache'
+      TMP_IMG_PATH = "#{TMP_BASE_PATH}tempFile.jpg"
```

Comment:
Will there be a collision if two user requests are writing at the same time?

Reply:
Good call, I would guess that would cause an issue. Would a single user ever be able to write to it twice at the same time? I could append the user id if they can't. Any other suggestions to get around this issue?

Reply:
Don't think a user would be able to write a the same time. But to be extra sure you could use a uuid (`SecureRandom.uuid`) for the file. Might want to create a directory for the user and just blow that away at the end, or on error, rather than have to clean up each file.

Reply:
Nice, that's a great idea. Thanks